user-interface/package.json

```diff
@@ -58,6 +58,13 @@
     "nth-check": "2.0.1",
     "webpack": "^5.78.0"
   },
+  "jest": {
+    "coveragePathIgnorePatterns": [
+      "/index.tsx",
+      "/reportWebVitals.ts",
+      "/configuration/apiConfiguration.ts"
+    ]
+  },
```

Comment:
I added index.tsx to the ignored patterns also, since it isn't doing anything logical, but rather just attaching the app to the root html element and rendering the base react App component.

It's also running reportWebVitals(), which sets up tracking for google analytics etc.